langchain4j-weaviate/pom.xml

```diff
@@ -36,6 +36,19 @@
 
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.honton.chas</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <configuration>
+                    <!-- the weaviate module has non-permissive licenses -->
```

Comment:
Internet says BSD 3-Clause is permissive, so better to update this comment.
(But let's not add it to the list of permitted licenses and keep this exclusion.)

Reply:
God point. Fixed the comment.